src/runtime/runtime-simd.cc

```diff
@@ -513,6 +513,31 @@ RUNTIME_FUNCTION(Runtime_##TYPE##Swizzle) {                   \
 SIMD128_SWIZZLE_FUNCTIONS(DECLARE_SIMD_SWIZZLE_FUNCTION)
 
 
+#define SIMD128_SWIZZLE_64x2_FUNCTIONS(V)                     \
```

Comment:
@huningxin Just for style consistency, if you think we don't need to do this, I will convert the template to instance.